### ISSUE TITLE
New place to chat?

### ISSUE THREAD

**Issue by naturallymitchell** [anowell/are-we-learning-yet#48]
https://blog.rust-lang.org/2019/04/26/Mozilla-IRC-Sunset-and-the-Rust-Channel.html

**Comment by anowell** [comment]
I'm not the right person to drive the new place to chat, but I'll absolutely support reflecting such change on the are-we-learning-yet site.

**Comment by naturallymitchell** [comment]
I see now the site has a Discord link.  But since it goes to the deprecated(??) Rust related programming server, and there's now an [official Rust Discord server](https://discord.gg/rust-lang), finding a channel there seems like a good start.  I couldn't immediately find an issue queue for managing the Discord server though, so if anyone can help get that rolling, that'd be cool.

**Comment by quietlychris** [comment]
A unofficial/applicant Working Group focused around the machine learning domain has been put together over the last few weeks! The group's repository can be found at [github.com/rust-ml](https://github.com/rust-ml), along with a Zulip chat channel [here](https://rust-ml.zulipchat.com). @anowell During our most recent meeting, this site came up as a commonly-seen resource for people looking into machine learning in Rust. @anowell Would you be open to accepting PRs from members of the Rust-ML group (and, of course, you and anyone is welcome to participate!)? Alternatively, if you have other things on your plate at the moment, would you potentially be open to adding a few other people as maintainers to help keep things up-to-date as the ecosystem evolves? I personally think this site has a lot of value, and would really enjoy seeing it continue to be a good resource for Rustaceans interested in getting into ML.

**Comment by anowell** [comment]
yes. I've been falling behind on PRs, the weekly cron to update the numbers has been broken for a bit, and catching up some random weekend has been on my todo list for at least a month, not counting the random r/rust links I've been meaning to update w.r.t. content. Let me make that change immediately so it doesn't fall through the cracks too.

I may also be open to entertaining the notion of transferring this repo to the rust-ml org. That would basically just leave me the bottleneck on the domain name.

**Comment by anowell** [comment]
Current rust-ml members all have collab invitations immediately. Happy to continue discussing ways to improve the collaboration to make this more of a community project, but hopefully that immediately allows filling in the gaps where I've dropped the ball. (and also gives me a bit of extra motivation to fix the CI issue)

**Comment by quietlychris** [comment]
That's awesome, thank you! Depending on how you feel about things moving forward, I don't think that there would be any problem on our end with taking ownership of the repository, but is totally up to you--I don't think there's any push in that regard one way or another on this end. We're really focusing on trying to work with the existing ecosystem and help out where/when we can. There's no push to take ownership of things at an organizational level, although it's something that we're willing to do if that's the action that would be most helpful. 

 I'll let people know on the chat, and hopefully we'll have some things to add to it soon! If there's anything that comes up  the meantime, just let me know on here or feel free to pull up the Zulip chat.

**Comment by quietlychris** [comment]
@anowell This might be a bit of a dumb question, but I don't have much experience running web-based applications that aren't local. Is there a particular way that I should be trying to re-generate the actual site data every time that a new pull request is merged? My local branch shows the updated versions when I run it on my own machine, but the actual arewelearningyet.com site doesn't seem to show the updates.

**Comment by anowell** [comment]
@quietlychris Not at all a dumb question. I just added an [explanation of publishing at the bottom of the README](https://github.com/anowell/are-we-learning-yet#publishing). Basically, I set CI long ago to [auto-publish changes](https://github.com/anowell/are-we-learning-yet/blob/7c6730cd7830114f2807901c63aa139f8a1ae422/.travis.yml#L10-L17) long ago. However, sometime in December (yikes, is that really 6 months ago already?), the Travis job started failing. It's been on my mind regularly via weekly email reminders that it's failing (which I do see). Somehow it took the extra human nudge (thanks 😉) to actually go fix it in 5218214e (Basically, Travis CI seems to have made a change affecting either the default ruby or bundler; I just upgraded ruby and bundler and it started working again. While I was at it, I upgraded Jekyll too).
